mobile/lib/modules/search/providers/people.provider.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/modules/home/ui/asset_grid/asset_grid_data_structure.dart';
+import 'package:immich_mobile/modules/search/services/person.service.dart';
+import 'package:immich_mobile/shared/models/asset.dart';
+import 'package:openapi/api.dart';
+
+final personAssetsProvider = FutureProvider.family
+    .autoDispose<RenderList, String>((ref, personId) async {
+  final PersonService personService = ref.watch(personServiceProvider);
+
+  var assets = await personService.getPersonAssets(personId);
+
+  if (assets == null) {
+    return RenderList.empty();
+  }
+
+  var convertToAsset = assets.map((e) => Asset.remote(e)).toList();
+
+  return RenderList.fromAssets(convertToAsset, GroupAssetsBy.auto);
```

Comment:
Do we want to use auto grouping always or use the user's setting?

Reply:
I don't have a reference, maybe user setting is good as well

Reply:
Let's keep it as is for now :)